src/test/java/org/folio/rest/impl/MockServer.java

```diff
@@ -297,6 +297,7 @@ private Router defineRoutes() {
     router.route(HttpMethod.DELETE, resourcePath(PO_LINES)).handler(ctx -> handleDeleteGenericSubObj(ctx, PO_LINES));
     router.route(HttpMethod.DELETE, resourcePath(ALERTS)).handler(ctx -> handleDeleteGenericSubObj(ctx, ALERTS));
     router.route(HttpMethod.DELETE, resourcePath(REPORTING_CODES)).handler(ctx -> handleDeleteGenericSubObj(ctx, REPORTING_CODES));
+    router.route(HttpMethod.DELETE, resourcePath(PIECES)).handler(ctx -> handleDeleteGenericSubObj(ctx, PIECES));
```

Comment:
shouldn't the route to storage for delete be set on the path {pieces_endpoint}/id ? so this will be resourcesPath(PIECES)+"/:id")

Reply:
That is being called here https://github.com/folio-org/mod-orders/blob/master/src/test/java/org/folio/rest/impl/MockServer.java#L700